bin/cli.mjs

```diff
@@ -77,6 +77,12 @@ program
       .choices(Object.keys(reporters))
       .default('console')
   )
+  .addOption(
```

Comment:
Instead of this, set a number of threads and can be set to 1 to disable parallelism. Call it `--threads`